os/board/rtl8721csm/src/component/os/tizenrt/rtk_netmgr.c

```diff
@@ -296,6 +297,7 @@ g_mode = RTK_WIFI_NONE;
 trwifi_result_e wifi_netmgr_utils_deinit(struct netdev *dev)
 {
 	trwifi_result_e wuret = TRWIFI_FAIL;
+	wifi_monitor_stop_timer();

```

Comment:
do we have to stop the timer only after successful cmd_wifi_off?

Reply:
Made it independent of wlan using the variable in the new implementation. So monitor will be stopped after successful cmd_wifi_off.